scripts/update-search-index.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
```

Comment:
I'll update this file name to be generate-search-index.sh once we remove the parts that are actually doing the update.